app.js

```diff
@@ -110,27 +114,67 @@ function getHashCode(str) {
   return hash;
 }
 
+function dedent(str) {
+  // Reduce the indentation of all lines by the indentation of the first line
+  const match = str.match(/^\n?( +)/);
+  if (!match) {
+    return str;
+  }
+  const indentRe = new RegExp(`\n${match[1]}`, 'g');
+  return str.replace(indentRe, '\n').replace(/^\n/, '');
+}
+
+function formatOptionValue(key, value) {
+  if (typeof value === 'string') {
+    return value;
+  }
+  if (CERT_OPTIONS.includes(key)) {
+    return chalk`${
+      value.toString()
+        .replace(/-----.+?-----|\n/g, '')
+        .substring(0, 80)
+    }{white …}`;
+  }
+  if (!value && value !== false) {
+    return UNDEFINED_VALUE;
+  }
+  if (typeof value === 'function') {
+    const lines = `${value}`.split('\n');
+    return lines[0].slice(0, -2);
+  }
+  return `${JSON.stringify(value)}`;
+}
+
+function prettyPrintXml(xml, indent) {
+  // This works well, because we format the xml before applying the replacements
+  const prettyXml = xmlFormat(xml, {indentation: '  '})
+    // Matches `<{prefix}:{name} .*?>`
+    .replace(/<(\/)?((?:[\w]+)(?::))?([\w]+)(.*?)>/g, chalk`<{green $1$2{bold $3}}$4>`)
+    // Matches ` {attribute}="{value}"
+    .replace(/ ([\w:]+)="(.+?)"/g, chalk` {yellow.bold $1}={blue "$2"}`);
+  if (indent) {
+    return prettyXml.replace(/(^|\n)/g, `$1${' '.repeat(indent)}`);
+  }
+  return prettyXml;
+}
+
 
 /**
  * Arguments
  */
 function processArgs(args, options) {
   var baseArgv;
-  console.log();
-  console.log('loading configuration...');
```

Comment:
This seemed unnecessary, so I removed it. I don't mind adding it back, if you disagree.

Reply:
i'm fine with it